src/features/surveys/l10n/messageIds.ts

```diff
@@ -94,6 +94,10 @@ export default makeMessages('feat.sruveys', {
     questions: m('Questions'),
     submissions: m('Submissions'),
   },
+  unlinkedCard: {
+    description: m('Submissions not linked to person records'),
+    header: m('Unlinked submitters'),
```

Comment:
I would like this text to be:

> Unlinked submissions

Reply:
Actually, I was a bit dubious when you asked me change unlinkedSubmitterCount to unlinkedSubmissionCount
Maybe should have asked you why, but anyway,
After I read the `description`. I think it makes more sense. Plus, it will have this link to submissions page so yeah, it is more logical to me